mesh/accountSettings/views.py

```diff
@@ -29,3 +33,52 @@ def display_theme(request):
                 response.update({"status": "error"})
                 response.update({"message": "An account does not exist with this account ID."})
                 return JsonResponse(response)
+
+class showSettings(View):
+    """ 
+    Handles HTTP requests related to Two Factor Authentication account settings.
+    Support Get request to retrieve setting and patch to change setting.
+    """
+    def get(self, request, account_id, *args, **kwargs):
+        """ 
+        Handles GET request
+
+        Return JSON with Two Factor Authentication Settings for given account ID. 
+        Returns a 404 error with error message if account does not exist.
+        """
+        try:
+            settings = Settings.objects.get(accountID=account_id)
+            settings_detail = serializers.serialize('json', [settings])
+            return JsonResponse(settings_detail, safe=False)
```

Comment:
I think here, instead of using JsonResponse, use `HttpResponse(settings_detail, content_type="application/json")` so that we don't double serialize our response data, as someone brought this up a week or two ago and it makes sense to just return it as HttpResponse instead.